Feature/add_about_myself_section

Shahid-Bagwan opened the pull request:
## Pull Request: Add About & Work_Section in HTML & CSS Files

### Changes Made


- Work Section: A new section has been added to the document specifically dedicated to showcasing project details. This section is designed to provide a clear and comprehensive overview of the work accomplished, allowing stakeholders to easily understand the progress and milestones achieved.

- About Section: To provide a concise and structured description of our skills, an "About" section has been incorporated into the document. This section now includes an unordered list (ul) format, which effectively highlights the skills possessed by our team members. This modification aims to present our expertise in a more accessible and organized manner, enabling our readers to quickly identify and assess our capabilities.

Please take note of these modifications and ensure that you utilize the updated version of the document moving forward. If you have any questions or require further clarification regarding these changes, please do not hesitate to reach out.

Thank you for your attention and cooperation.

Shahid-Bagwan committed:
Update Index & Style file.

the following changes are made :

- add work section: inserted box where project details can be shown.

- add about section: made a ul list in which skills are mentioned.

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
-            <div class="image_container">
-                <img src="resources/Illustration.png" alt="image">
+        </section>
+        <div>
+            <div class="work_section">
```
- [ ] Please 🙏🏽 use the `section` tag to content the work section for the better practices of HTML & CSS:

Shahid-Bagwan replied:
@franclobo Done!

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
         </div>
-    </section>
+
+        <div class="about">
```
- [ ] Please 🙏🏽 use the `section` tag to content the about section for the better practices of HTML & CSS:

Shahid-Bagwan replied:
@franclobo Done!

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+        </section>
+        <div>
+            <div class="work_section">
+                <div class="work_section_title"> <img src="resources/Title.png" alt=""></div>
```
- [ ] The works section title is not an image, please 🙏🏽 use the correct `hx` tag and implement the correct styles to be similar to the Figma template.

Shahid-Bagwan replied:
@franclobo Done!

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+            <div class="work_section">
+                <div class="work_section_title"> <img src="resources/Title.png" alt=""></div>
+                <div class="first_box">
+                    <img src="resources/Img Placeholder.png" alt="">
```
- [ ] Kindly add a description of the image in the `alt` attribute for the best practices of HTML & CSS.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="first_box_lang">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/css.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                        Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="lang_used">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                        Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="lang_used">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                        Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="lang_used">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                        Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="lang_used">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                        Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="lang_used">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                        Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <div class="lang_used">
+                        <img src="resources/html.png" alt="">
+                        <img src="resources/bootstrap.png" alt="">
+                        <img src="resources/ruby.png" alt="">
+                    </div>
```
- [ ] The list of features is not images please 🙏🏽 use the correct `ul` and `li` tags to list them and add the correct styles, border solid, font styles, and correct margin and padding to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+            <div class="skills">
+
+                <div class="check">
+                    <img src="resources/Frame 30.png" alt="">
```
- [ ] This is a title of a list, please 🙏🏽 use the correct `hx` tag and add the correct styles to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                    </ul>
+                </div>
+                <div class="check">
+                    <img src="resources/Frame 31.png" alt="">
```
- [ ] This is a title of a list, please 🙏🏽 use the correct `hx` tag and add the correct styles to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on index.html:
```diff
@@ -15,33 +15,183 @@
+                    </ul>
+                </div>
+                <div class="check">
+                    <img src="resources/Frame 32.png" alt="">
```
- [ ] This is a title of a list, please 🙏🏽 use the correct `hx` tag and add the correct styles to be similar to the Figma template.

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    margin-left: -14%;
+  }
+
+  .first_box_button {
+    margin-right: auto;
+    margin-top: -2%;
+    background: #ff6b00;
+    width: 34%;
+    height: 6%;
+    color: white;
+    margin-left: 6%;
+  }
```
- [ ] Kindly add the selectors, hover, enabled, pressed, and disabled as specified in the Figma design, and please 🙏🏽 correct the styles of the button to be similar to the original design. 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/0853ff0a-4ac0-4689-97ed-90e8453ba58b)

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/323d3f50-2b4d-489a-b868-29d0f5d3fdd1) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/de3a9d8f-44f8-4a90-8107-3c2d6a427daa) |

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    margin-bottom: -20%;
+  }
+
+  .box button {
+    margin-top: auto;
+    font-family: 'Inter', sans-serif;
+    font-weight: bold;
+    font-size: 17px;
+    background: #ff6b00;
+    width: 100%;
+    height: 3rem;
+    color: white;
+  }
```
- [ ] Kindly add the selectors, hover, enabled, pressed, and disabled as specified in the Figma design, and please 🙏🏽 correct the styles of the button to be similar to the original design. 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/d3a44f67-a010-4a0c-8204-e1d512cdaaae) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/0911b372-e38d-477f-866a-0e2b1b91c399) |

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    margin: 0 0 0 0;
+  }
+
+  .first_box h2 {
+    font-size: 1.5rem;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 100;
+    margin: 4% auto 0 7%;
+  }
```
- [ ] Please 🙏🏽 add the correct styles to be similar to the Figma design 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/fd7f5070-0622-4461-a9af-a4c7cc85efe7)

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    margin: 4% auto 0 7%;
+  }
+
+  .first_box p {
+    width: 88%;
+    padding-left: 1%;
+  }
```
- [ ] Please 🙏🏽 add the correct styles to be similar to the Figma design 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/a04efab6-6642-400e-8ad2-b60f92ac1a02)

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    padding-left: 1%;
+  }
+
+  .first_box_lang {
+    display: flex;
+    flex-flow: row;
+    justify-content: space-around;
+    margin-top: 1%;
+    margin-bottom: 4%;
+    margin-left: -14%;
+  }
```
- [ ] Please 🙏🏽 add the correct styles, margin, and padding to be similar to the Figma design 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/323d3f50-2b4d-489a-b868-29d0f5d3fdd1) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/de3a9d8f-44f8-4a90-8107-3c2d6a427daa) |

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    padding-top: 10%;
+  }
+
+  .box h2 {
+    font-size: 1.5rem;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 100;
+    margin: 32% 0 0 5%;
+  }
```
- [ ] Please 🙏🏽 add the correct styles to be similar to the Figma design 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/05a10195-ed4d-498f-b8a3-734f683629c7)

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    margin: 32% 0 0 5%;
+  }
+
+  .box p {
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    width: 88%;
+    padding-left: 5%;
+  }
```
- [ ] Please 🙏🏽 add the correct styles to be similar to the Figma design 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/d869f180-28fb-42d8-8df5-ae1dbb032227)

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    padding-left: 5%;
+  }
+
+  .lang_used {
+    display: flex;
+    flex-flow: row;
+    justify-content: space-around;
+    margin-top: 5%;
+    margin-bottom: -20%;
+  }
```
- [ ] Please 🙏🏽 add the correct styles, margin, padding, and background color to be similar to the Figma design 👇🏽

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/d3a44f67-a010-4a0c-8204-e1d512cdaaae) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/0911b372-e38d-477f-866a-0e2b1b91c399) |

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    height: 26.188rem;
+  }
+
+  .h1_about {
+    font-size: 3.5rem;
+    color: #ff6b00;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 100;
+    margin: 0 0 10px 16px;
+    width: fit-content;
+  }
```
- [ ] Please 🙏🏽 add the correct styles to be similar to the Figma design 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/5ffa7a05-45cb-47ca-bbdb-9ac2c72d1bd3)

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    width: fit-content;
+  }
+
+  .about_paragraph {
+    font-size: 1rem;
+    font-family: 'Inter', sans-serif;
+    word-wrap: break-word;
+    color: white;
+    margin: 0 0 0 16px;
+    width: 375px;
+  }
```
- [ ] Please 🙏🏽 add the correct styles, margin, and padding to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/1d0c1192-1f11-479b-ae35-7e21a7279f20) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/c5857d9b-cff6-4cda-aa49-17ae028a41d5) |

Shahid-Bagwan replied:
Done!

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    width: 375px;
+  }
+
+  .about_box_button {
+    margin-top: 5%;
+    font-family: 'Inter', sans-serif;
+    font-weight: bold;
+    font-size: 17px;
+    background: #ff6b00;
+    width: 43%;
+    height: 14%;
+    color: white;
+    margin-left: 3%;
+  }
```
- [ ] Kindly add the selectors, hover, enabled, pressed, and disabled as specified in the Figma design, and please 🙏🏽 correct the styles of the button to be similar to the original design. 👇🏽 

![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/0853ff0a-4ac0-4689-97ed-90e8453ba58b)

Shahid-Bagwan replied:
Done

franclobo commented on style.css:
```diff
@@ -97,4 +99,210 @@ ul {
+    padding-top: 40%;
+  }
+
+  .check {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+  }
```
- [ ] Kindy center the text to be similar to the Figma design 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/f9022e87-9e54-47ad-9b53-282dad5d32f4) | ![image](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/ccdb2148-1b06-40e3-b7ca-945b68f97a57) |

Shahid-Bagwan replied:
Done!

franclobo reviewed (changes requested):
Hi @Shahid-Bagwan, 

# Required Changes ♻️

![so-close-jim-halpert](https://github.com/Shahid-Bagwan/Portfolio/assets/58642949/f389dfd8-b81c-483b-9972-a495c08d0227)

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- COrrect Github flow ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Update index & style files.

Made changes Suggested by Reviewer :

- add font, font-size, font-weight, font-style, line-height.

- add button hover, active, disable.

- add space according to the desing & code reviewr.

- replace button images with buttons.

Shahid-Bagwan committed:
Fix : Webhint errors

Fix the following Webhint errors:

- add type attribute in button.

- remove inline css.

Abenezer-Tilahun commented on index.html:
```diff
@@ -11,37 +11,203 @@
+
+        <section class="about">
+            <div class="about_section">
+                <h1 class="h1_about">About me</h1>
```
- [ ] Kindly I recommend you to use `hx (h2 )` tag for the **About me** heading for the purpose optimization and search engine and to follow the best practice for HTML and CSS .

Shahid-Bagwan replied:
@Abenezer-Tilahun Done!

Abenezer-Tilahun commented on index.html:
```diff
@@ -11,37 +11,203 @@
+        </section>
+        <div class="Second_part">
+            <section class="work_section">
+                <div class="work_section_title">My Recent Works </div>
```
- [ ] Kindly I recommend you to  uses a header tag `hx(h2)` tag for the title (e.g., "My Recent Works") to follow the Best Practice of HTML and CSS.

Shahid-Bagwan replied:
@Abenezer-Tilahun Done!

Abenezer-Tilahun commented on style.css:
```diff
@@ -97,4 +116,328 @@ ul {
+    cursor: not-allowed;
+  }
+
+  /* about section */
+  .about {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    background-color: #1c1a19;
+    height: 100%;
+    row-gap: 1.625rem;
+    margin-bottom: 0;
+  }
+
+  .about_section {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    height: 26.188rem;
+    margin: 4.75rem 0 0 1.188rem;
+  }
```
- [ ] As stated in the project requirement Make sure the content of the "About myself" section is aligned with `flexbox. `So for the about section kindly update the section aligned with `flexxbox `instead of the `grid.`

Shahid-Bagwan replied:
@Abenezer-Tilahun Done!

Abenezer-Tilahun reviewed (changes requested):
Hi @Shahid-Bagwan  , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Good commit messages
✔️ Professional README

![almost](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Required Changes ♻️
_Check the comments below the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shahid-Bagwan committed:
Update Index & Style

Made Change Suggested by Code Reviewer :

- add h2 tags.

- replace display grid with flex.

nahnahdev reviewed (approved):
## Project Status: APPROVED :green_circle: 
Hi @Shahid-Bagwan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://media0.giphy.com/media/3oz9ZE2Oo9zRC/giphy.gif"/>

## Highlights 🔥 
- All requested changes implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
